f2/src/forms/HowDidItStartForm.js

```diff
@@ -144,90 +143,50 @@ export const HowDidItStartForm = props => {
               })}
             </FormArrayControl>
 
-            <Stack>
-              <P fontWeight="bold">
-                <Trans id="whenDidItStart.label" />
-              </P>
-              <P fontSize="md">
-                <Trans id="whenDidItStart.labelExample" />
-              </P>
-            </Stack>
```

Comment:
Replacing this with `<FormArrayControl>` 
this Returns a fieldset and a legend. All children will be part of the same group. See this page for reference: [WAI grouping](https://www.w3.org/WAI/tutorials/forms/grouping/)

We use this for checkboxes and radio lists, but we can also use it to group multiple fields together like for the date fields